app/views/talks/_talk.html.erb

```diff
@@ -43,14 +43,36 @@
     <div class="flex flex-col gap-1">
       <div class="flex justify-between">
         <div class="font-bold text-xl">
-          <%= talk.title %>
+          <span><%= talk.title %></span>
         </div>
 
-        <div class="tooltip tooltip-top" data-tip="You can suggest some modifications to this talk">
-          <%= link_to edit_talk_path(talk), class: "flex gap-2 items-center rounded-full border bg-gray-100 hover:bg-gray-200/50 border-gray-200 font-regular px-3 py-1 text-sm" do %>
-            <%= heroicon :pencil, size: :sm %>
-            <span>Edit</span>
+        <div class="flex gap-1">
+          <% if signed_in? %>
+            <% if talk.watched? %>
+              <div class="tooltip tooltip-top" data-tip="Mark talk as unwatched">
+                <%= button_to [talk, :watched_talk],
+                      class: "flex gap-2 items-center rounded-full border bg-gray-100 hover:bg-gray-200/50 border-gray-200 font-regular px-3 py-1 text-sm",
+                      method: :delete do %>
+                  <%= heroicon :x_mark, size: :sm %>
```

Comment:
I somehow feel like the `x` is weird here, since you "unwatch" is usually not a typical action.

Reply:
@marcoroth  what do you think of something like this?
![Kapture 2024-11-11 at 07 41 42](https://github.com/user-attachments/assets/defd96a3-5141-425e-b429-a5bc2c14b5a2)

Reply:
I personally like this a lot better, thanks for exploring 🙌🏼

Reply:
Thinking about it again, I wonder if the text should even change when the state changes.

What if you have an empty circle for the "unwatched" state that gets checked off if you mark it as "watched" but the text always says "Watched"?

Reply:
@marcoroth I first looked for an empty circle on the [heroicons list](https://heroicons.com) but couldn’t find one, so I ended up going with `minus-circle`.

Here we have the two versions.
![Kapture 2024-11-11 at 08 09 12](https://github.com/user-attachments/assets/b37c5e95-9e43-4457-ba12-590f247739fa)

![Kapture 2024-11-11 at 08 13 48](https://github.com/user-attachments/assets/a751e68a-67e4-4066-ab1d-c74eed189ee8)

I'm not sure which one works better. What do you think?